### PR TITLE
[Keymap] plaid keymap with custom LED animation

### PR DESCRIPTION
**Pull request by gipsy-king**
## Description

My keymap for the plaid keyboard, with custom LED animation

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dm9records/plaid/keymaps/gipsy-king/keymap.c [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (drashna) [inline keyboards/dm9records/plaid/keymaps/gipsy-king/keymap.c:19]: This isn't needed and should be removed. 


```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/dm9records/plaid/keymaps/gipsy-king/keymap.c (hunk 1 above)
Comment (gipsy-king) [reply keyboards/dm9records/plaid/keymaps/gipsy-king/keymap.c]: removed.

**Review by drashna — APPROVED** [review]

File: keyboards/dm9records/plaid/keymaps/gipsy-king/config.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* Copyright 2019 Takuya Urakawa (dm9records.com)
```
Comment (fauxpark) [inline keyboards/dm9records/plaid/keymaps/gipsy-king/config.h:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/dm9records/plaid/keymaps/gipsy-king/config.h (hunk 2 above)
Comment (gipsy-king) [reply keyboards/dm9records/plaid/keymaps/gipsy-king/config.h]: fixed, sorry for the inconvenience.
also spotted some missing keys (brackets) and fixed some alignment.

**Pull request merged**